pcc/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@pcc/starter-kit",
+  "name": "@pantheon-systems/pcc",
```

Comment:
If app name is `pcc` then probably we can remove `bin` entry from this file

Reply:
Hm, you're right, but also seems harmless to keep it.